core/services/relay/evm/evm.go

```diff
@@ -345,7 +345,7 @@ func newContractTransmitter(lggr logger.Logger, rargs relaytypes.RelayArgs, tran
 		gasLimit,
 		effectiveTransmitterAddress,
 		strategy,
-		txm.EvmTransmitCheckerSpec{},
```

Comment:
this was ignored the constructed checker above preventing using the sim flag in OCR2